api/Insight_API_documentation.md

```diff
@@ -582,7 +581,7 @@ Methods that work with addresses.
 
 **URL:**  ```GET /addr/{address}/totalSent```
 
-**Description:** Retrieves total amount sent from an address. **REVIEW. Is it from?** 
+**Description:** Retrieves total amount sent from an address in atoms (the smallest unit of Decred; 1 DCR = 100,000,000 atoms)
```

Comment:
The whole "from an address" thing is very contentious (rightly), but given the endpoint path, I think to be clear we have to use this wording.  I just want to note that a more precise explanation of this amount might be "the total value of all spent transaction outpoints paying to the given address".

Reply:
Addresses are used with transaction outputs, so they are destinations for funds being sent, although this endpoint is saying "from an address."    When you send funds to an address with some transaction, it creates a transaction output that when unspent is an "unspent transaction output" (UTXO). Since addresses are reusable, you can create more transactions that pay to the address, and each of those transactions creates new outputs.

Since an input to a transaction is matched to exactly one previous transaction output (except coinbase/stakebase inputs), the UTXOs get spent and are spent transaction outputs (STXOs).  When people say "from an address" they mean a transaction consumed (as an input) another transaction's output that payed to this address.

For `totalSent`, the wording "send by an address" or "sent from an address" are referring to the sum of all STXOs that had paid to the address, and are now spent.
For `totalReceived`, this is the sum of all UTXOs and STXOs.
For `balance`, this is the sum of all UTXOs.

i.e. `totalReceived` -`totalSent` = `balance`

Reply:
@s-ben I think @chappjc is agreeing that the current wording is acceptable, but he is offering a more technically accurate version.  They are our docs and we can do what we want with them, so why not just give the world a little  more detail:

### /addr/ (totalSent)
 **URL:**  ```GET /addr/{address}/totalSent```
 **Description:** Retrieves total amount sent from an address in atoms (the smallest unit of Decred; 1 DCR = 100,000,000 atoms)

>**Developer Note:**  Addresses are used with transaction outputs, so they are destinations for funds being sent, although this endpoint is saying "from an address."    When you send funds to an address with some transaction, it creates a transaction output that when unspent is an "unspent transaction output" (UTXO). Since addresses are reusable, you can create more transactions that pay to the address, and each of those transactions creates new outputs.
>
>Since an input to a transaction is matched to exactly one previous transaction output (except coinbase/stakebase inputs), the UTXOs get spent and become spent transaction outputs (STXOs).  When people say "from an address" they mean a transaction consumed (as an input) another transaction's output that payed to this address.
>
>For `totalSent`, the wording "send by an address" or "sent from an address" are referring to the sum of all STXOs that had paid to the address, and are now spent.
>For `totalReceived`, this is the sum of all UTXOs and STXOs.
>For `balance`, this is the sum of all UTXOs.
>
>i.e. `totalReceived` -`totalSent` = `balance`